pallets/xyk/src/mock.rs

```diff
@@ -66,9 +68,14 @@ impl orml_tokens::Trait for Test {
     type WeightInfo = ();
 }
 
+parameter_types! {
+    pub const NativeCurrencyId: u32 = NATIVE_CURRENCY_ID;
```

Comment:
now i think first try to mint new currency will fail - as it takes id from the Tokens storage and its default value is `0`. Can we somehow provide proper initial value (`1`) to tokens storage(`NextCurrencyId`) ?